be/src/vec/exprs/vbitmap_predicate.cpp

```diff
@@ -90,14 +87,14 @@ doris::Status vectorized::VBitmapPredicate::execute(vectorized::VExprContext* co
         arguments[i] = column_id;
     }
     // call function
-    size_t num_columns_without_result = block->columns();
+    uint32_t num_columns_without_result = block->columns();
     auto res_data_column = ColumnVector<UInt8>::create(block->rows());
 
     ColumnPtr argument_column =
             block->get_by_position(arguments[0]).column->convert_to_full_column_if_const();
     size_t sz = argument_column->size();
     res_data_column->resize(sz);
-    auto ptr = ((ColumnVector<UInt8>*)res_data_column.get())->get_data().data();
+    auto* ptr = ((ColumnVector<UInt8>*)res_data_column.get())->get_data().data();
```

Comment:
why type conversion here
```suggestion
    auto* ptr = ((ColumnVector<UInt8>*)res_data_column.get())->get_data().data();
```